docs/src/examples/components/Button/Types/ButtonExampleIconOnly.shorthand.tsx

```diff
@@ -1,6 +1,17 @@
 import * as React from 'react'
-import { Button } from '@stardust-ui/react'
+import { Button, Flex, Text } from '@stardust-ui/react'
 
-const ButtonExampleIconOnly = () => <Button icon="book" iconOnly primary />
+const ButtonExampleIconOnly = () => (
+  <div>
+    <Flex gap="gap.large" vAlign="center">
+      <Button icon="stardust-close" iconOnly />
+      <Text content="AS A DEFAULT BUTTON" weight="bold" />
+    </Flex>
+    <Flex gap="gap.large" vAlign="center">
+      <Button icon="stardust-close" text iconOnly />
+      <Text content="AS A TEXT BUTTON" weight="bold" />
```

Comment:
Wouldn't it be better to have all text variations in one example (default, primary, iconOnly etc...) What was the reason these were separated across different examples?

Reply:
Yes, good callout. I added them.